internal/prober/testdata/config_missing_optional_values.yaml

```diff
@@ -5,7 +5,6 @@ dependentResourceInfos:
       kind: "Deployment"
       name: "kube-controller-manager"
       apiVersion: "apps/v1"
-    optional: false
```

Comment:
The name of the file says it is for a config with missing optional values. A bit confusing as `optional` is both a field name in a struct and also it could also mean configuration where all optional values are absent.